gamma/1.4/Dockerfile

```diff
@@ -0,0 +1,84 @@
+# base image
+FROM ubuntu:bionic
+
+# metadata
+LABEL base.image="ubuntu:bionic"
+LABEL container.version="1"
+LABEL software="centroid"
+LABEL software.version="v1.4"
+LABEL description="Gene Allele Mutation Microbial Assessment"
+LABEL website="https://github.com/rastanton/GAMMA/tree/v1.4"
+LABEL license=""
```

Comment:
Maybe put none for license? GAMMA GitHub repo doesn't show a license but the [bioconda page says Apache 2.0](https://bioconda.github.io/recipes/gamma/README.html) 

We could ask Rich to add a license to the GH repo